OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -144,7 +144,12 @@ void INotifyCreated.Created(Actor self)
 			conditionManager = self.Trait<ConditionManager>();
 		}
 
-		static int GetWeight(Actor a) { return a.Info.TraitInfo<PassengerInfo>().Weight; }
+		public int GetWeight(Actor a) { return GetPassengerTrait(a).Info.Weight; }
+
+		public Passenger GetPassengerTrait(Actor a)
```

Comment:
Please explain me how write a lint test. The best way send me link to similar code in the project.